src/js/components/TextInput/__tests__/TextInput-test.js

```diff
@@ -155,6 +155,63 @@ describe('TextInput', () => {
     }, 50);
   });
 
+  test('calls onSuggestionsOpen', done => {
+    const onSuggestionsOpen = jest.fn();
+    const { getByTestId } = render(
+      <Grommet>
+        <TextInput
+          data-testid="test-input"
+          id="item"
+          name="item"
+          suggestions={['test', 'test1']}
+          onSuggestionsOpen={onSuggestionsOpen}
+        />
+      </Grommet>,
+    );
+
+    fireEvent.focus(getByTestId('test-input'));
+    fireEvent.change(getByTestId('test-input'), { target: { value: ' ' } });
```

Comment:
I'm thinking this call might be redundant, `expect(onSuggestionsOpen).toBeCalled();` should happen just by focus and not necessarily by a change event. If it tests something else we can put it on a separate test case.

Reply:
Yes, you're right. Just focus is sufficient.